src/lib/core/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set(core_sources
     mp_datetime.c
     mp_interval.c
     prbuf.c
+    clock_lowres.c
```

Comment:
We actually have a `clock.c`: perhaps move your code there?

@locker RFC

Reply:
I'm okay either way. Having `clock_lowres()` along with `clock_monotonic()` looks good though.

Reply:
Other functions from `clock.c` are not so limited (`clock_lowres` can be called only from main thread). Is it good idea to have such functions in one module?

Reply:
`clock_lowres()` may be called from any thread, I think, see https://github.com/tarantool/tarantool/pull/7160#discussion_r878246693

Reply:
I don't want to put work with threads in the `clock.c` file, so I decided to leave it in a separate file.